packet/baseapi.py

```diff
@@ -5,6 +5,8 @@
 import logging
 import requests
 
+from Packet import __version__
```

Comment:
Your tests are failing because the casing of this import is incorrect.

```suggestion
from packet import __version__
```